app/src/main/java/com/techcourse/service/TxUserService.java

```diff
@@ -29,15 +29,15 @@ public User findById(final long id) {
 
     @Override
     public void insert(final User user) {
-        transactional(() -> userService.insert(user));
+        executeInTransaction(() -> userService.insert(user));
     }
 
     @Override
     public void changePassword(final long id, final String newPassword, final String createBy) {
-        transactional(() -> userService.changePassword(id, newPassword, createBy));
+        executeInTransaction(() -> userService.changePassword(id, newPassword, createBy));
     }
 
-    private void transactional(final TransactionalExecutor executor) {
+    private void executeInTransaction(final TransactionalExecutor executor) {
         final TransactionStatus transactionStatus = transactionManager.getTransaction(new DefaultTransactionDefinition());
```

Comment:
개인적으로 `executeTransaction`이라고 해도 괜찮을 것 같아요.

```suggestion
    private void executeTransaction(final TransactionalExecutor executor) {
        final TransactionStatus transactionStatus = transactionManager.getTransaction(new DefaultTransactionDefinition());
```

Reply:
감사합니다!!